hsmd/libhsmd.c

```diff
@@ -1257,6 +1259,22 @@ static u8 *handle_sign_commitment_tx(struct hsmd_client *c, const u8 *msg_in)
 	return towire_hsmd_sign_commitment_tx_reply(NULL, &sig);
 }
 
+/* This handler is for signers which need to independently validate
```

Comment:
comment is a bit confusing.  we mean to say that a fully validating `hsmd` would override this stub implementation so that it can independently validate